### PR TITLE
Fixes for Daydream

### PR DESCRIPTION
**Pull request by robertlong**
+ Fixes a bug that @johnshaughnessy and I found with input mapping events and the `virtual-gamepad-controls` component.
+ Adds initializing the `daydream-controls` component to `hand-controls2` component.
+ Uses a custom version of AFrame that sets the correct camera height when entering VR on mobile devices (See https://github.com/aframevr/aframe/pull/3111).
+ Adds an `offline=true` query string option so you can get past the loading screen when offline.
+ Fixes a typo with the avatar body mesh.

### CODE REVIEW

File: public/room.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 <head>
     <title>Mozilla Mixed Reality Social Client</title>
+    <script src="https://webrtc.github.io/adapter/adapter-latest.js"></script>
```
Comment (brianpeiris) [inline public/room.html]: Should we really point to `latest`?

File: src/components/character-controller.js [hunk 2]
```diff
@@ -116,15 +116,27 @@ AFRAME.registerComponent("character-controller", {
   },
 
   onTranslateX: function(event) {
-    this.accelerationInput.setX(event.detail);
+    if (typeof event.detail !== "object") {
+      this.accelerationInput.setX(event.detail);
+    } else {
+      this.accelerationInput.setX(0);
+    }
```
Comment (brianpeiris) [inline src/components/character-controller.js]: Is this a bug in input-mapping that we should submit a PR for? Maybe put a TODO in here.

File: src/components/hand-controls2.js [hunk 3]
```diff
@@ -106,6 +106,7 @@ AFRAME.registerComponent("hand-controls2", {
       el.setAttribute("vive-controls", controlConfiguration);
```
Comment (brianpeiris) [inline src/components/hand-controls2.js]: (why is this component called `hand-controls2`? Surely it deserves a better name)

Comment (netpro2k) [reply src/components/hand-controls2.js]: I think its a direct extension of the aframe built in component `hand-controls` . It's likely something we might want to land upstream but we don't know exactly what we want yet so its not worth maintaining a seperate fork of aframe over. Thus `hand-controls2` was born.

**Pull request merged**